scala/scala_3/build.gradle

```diff
@@ -0,0 +1,21 @@
+dependencies {
+    implementation 'org.scala-lang.modules:scala-java8-compat_3'
+
+    // Added for supporting Scala types in Jackson{Request,Response}ConverterFunction
+    implementation 'com.fasterxml.jackson.module:jackson-module-scala_3'
+}
+
+// Use the sources from ':scala_2.13'.
```

Comment:
Shouldn't we move sources to scala_3 and copy them to scala_2.13?

Reply:
Why don't we make Scala 3 main version when https://github.com/lampepfl/dotty/issues/11332 is fixed and clean up the ScalaPB module?
I think Scala 3 is still experimental and we need to wait for it to be stabilized.

Reply:
SGTM. 😉